### PR TITLE
Liweip deb

### PR DESCRIPTION
**Pull request by LiweiPeng**

### CODE REVIEW

File: src/fluent-plugin-mdsd/outmdsd.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
     s.name = "fluent-plugin-mdsd"
-    s.version = "0.1.0"
+    s.version = "GEMVERSION"
```
Comment (LiweiPeng) [inline src/fluent-plugin-mdsd/outmdsd.gemspec]: this is for PR test purpose

**Comment by LiweiPeng** [comment]
>     s.license = "Microsoft Internal Only"

This needs to be updated.

---
Refers to: src/fluent-plugin-mdsd/outmdsd.gemspec:15 in 2961036. [](commit_id = 29610360720949c6f5db80e14f8f67cdba775e79, deletion_comment = False)

File: src/debpkg/control [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Package: PACKAGE
+Version: VERSION
+Section: libdevel
+Priority: optional
+Architecture: amd64
+Depends: DEPENDS
+Maintainer: Azure Linux Team <azlinux@microsoft.com>
+Description: Azure Linux monitoring agent (mdsd) output plugin for fluent
```
Comment (hosungsmsft) [inline src/debpkg/control]: >fluent [](start = 67, length = 6)

"fluentd." ?

File: src/debpkg/Makefile [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+clean: install-clean
+	-rm -rf control-root
+	-rm -f debian-binary *.tar.gz _gpgorigin md5sums
+	-rm -f lib*deb
+
+install-clean:
+	-rm -rf $(FAKEROOT)
+
+ifeq ($(CC), clang)
```
Comment (hosungsmsft) [inline src/debpkg/Makefile]: >clang [](start = 13, length = 5)

Do we build a clang version of this plugin as well?

**Comment by hosungsmsft** [comment]
:shipit:

File: src/debpkg/Makefile (hunk 3 above)
Comment (LiweiPeng) [inline src/debpkg/Makefile]: no, should be removed.

---
In reply to: [95899321](https://github.com/Azure/fluentd-plugin-mdsd/pull/1#discussion_r95899321) [](ancestors = 95899321)

**Pull request merged**